.gitignore

```diff
@@ -1,5 +1,6 @@
 #Java
 target/
+dependency-reduced-pom.xml
```

Comment:
Not needed?

Reply:
Maven generates it automatically every time we build the project.